exawind/NREL_5MW_Turbine/flexible/README.md

```diff
@@ -1,85 +1,87 @@
 <!-- This file is automatically compiled into the website. Please copy linked files into .website_src/ paths to enable website rendering -->
 
-<style>
-    img,
-    .whitebackground img {
-        color: #ffffff !important;
-    }
-
-    h1,h2,h3 {
-        margin-top: 30px;
-    }
-</style>
-
 # NREL 5MW Rotor in Atmospheric Boundary Layer With OpenFAST/BeamDyn Fluid Structure Interaction
 This benchmark contains a geometry and boundary-layer resolved model of the NREL 5MW reference turbine [1] in a developed near-neutral convective atmospheric boundary layer with a mean hub-height streamwise velocity of 11.4m/s.
 
 - Hub-height: 90m
 - Blade Radius: 63m
 - Rated Power: 5 MW
 - Type: Upwind 3 Blade
+- Simulation dt: 0.003443526170799 s
+- OpenFAST dt: 0.0008608815426997245 s
 
-<i>Note on run-time: Current results are for 125s simulation time, but by the time of bechmark release, a minimum of 360s will be posted.</i>
+<i>Note on run-time: Current results are for approximately 120s simulation time, but a minimum of 300s will be posted as results become available.</i>
 
 ## Simulation Setup
 - ExaWind driver SHA: [cba5259fc43ddeca67329630d3c84faad90e91bb](https://github.com/Exawind/exawind-driver/commit/cba5259fc43ddeca67329630d3c84faad90e91bb)
 - Nalu-Wind SHA: [b9e4ae654b646ecd0501dd6391dc7537239c82db](https://github.com/Exawind/nalu-wind/commit/b9e4ae654b646ecd0501dd6391dc7537239c82db)
 - AMR-Wind SHA: [091b07fa3840af98925792290fb8788696290a5e](https://github.com/Exawind/amr-wind/commit/091b07fa3840af98925792290fb8788696290a5e)
-- OpenFAST 4.0 SHA: [da685d4997fd17ea845812c785325efa72edcf47](https://github.com/OpenFAST/openfast/commit/da685d4997fd17ea845812c785325efa72edcf47)  
+- OpenFAST 4.0.2 SHA: [fc1110183bcc87b16d93129edabdce6d30e3a497](https://github.com/OpenFAST/openfast/commit/fc1110183bcc87b16d93129edabdce6d30e3a497)  
 
 ## Freestream Conditions
 A near-neutral convective boundary layer precursor was run to feed the domain for this case. Full details are posted here: [Convective ABL for NREL5MW](../../../amr-wind/atmospheric_boundary_layer/convective_abl_nrel5mw/README.md)
 
 10 minute flow statistics:
-- Hub-height streamwise velocity: 11.63 m/s
+- Hub-height streamwise velocity: 11.4 m/s
 - Hub-height flow direction: ~240.0 degrees
 
 <img src="../../../amr-wind/atmospheric_boundary_layer/convective_abl_nrel5mw/results/avgmetmast_0600.png" alt="10-Minute Average Virtual Met-mast Precursor Results" style="width:800px; background-color: #ffffff;"/>
```

Comment:
There's a limitation in the `deploypage.py` script, we need to use markdown-style image includes, so we need to switch things to this syntax:
```markdown
![avgmetmast](../../../amr-wind/atmospheric_boundary_layer/convective_abl_nrel5mw/results/avgmetmast_0600.png)
```
Sorry about that, that was my doing.

Reply:
This has been changed, but maybe I want to change image sizes depending on how it renders